model-optimizer/extensions/front/tf/if_ext.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from extensions.ops.If import If
+from extensions.ops.parameter import Parameter
+from mo.front.common.register_custom_ops import check_for_duplicates
+from mo.front.extractor import FrontExtractorOp, extract_node_attrs
+from mo.front.tf.extractor import tf_op_extractor, tf_op_extractors
+from mo.front.tf.extractors.subgraph_utils import update_body_graph, convert_graph_inputs_to_parameters, \
+    get_graph_proto, create_internal_graph
+from mo.graph.graph import Node, Graph
+
+
+def extract_if(cls, if_node: Node):
+    If.update_node_stat(if_node, {})
+
+    # check that required body and condition functions exist in the graph library
+    main_graph = if_node.graph
+    then_graph_proto = get_graph_proto(main_graph, 'then_branch', if_node)
+    else_graph_proto = get_graph_proto(main_graph, 'else_branch', if_node)
+
+    then_graph = create_internal_graph(main_graph)
+    if_node['then_graph'] = then_graph
+
+    else_graph = create_internal_graph(main_graph)
+    if_node['else_graph'] = else_graph
```

Comment:
why did you pass the same argument to `create_internal_graph` for creating both graphs?